ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -2674,6 +2683,7 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe
     env:
+      FEATURE_GATES: '[ClusterAPIInstall=true]'
```

Comment:
Once `TechPreviewNoUpgrade` is enabled, all the feature set will be enabled, no need to set `FEATURE_GATES` separately.

Reply:
BTW, [`ClusterAPIInstall`](https://github.com/openshift/api/blob/master/features/features.go#L349) is not enabled in the TechPreviewNoUpgrade feature gate by default, so when TP is enabled, it still would be a Terraform install, it's just fine to leave the TP test as before, we don't need to test CAPI with TP on purpose. 

Generally,  feature gate `ClusterAPIInstallIBMCloud` would be promoted to TechPreviewNoUpgrade and Default later, so we can test CAPI with TP and by default correspondingly.